source/views/dictionary/detail.tsx

```diff
@@ -26,37 +29,46 @@ const styles = StyleSheet.create({
 })
 
 type Props = TopLevelViewPropsType & {
-	navigation: {state: {params: {item: WordType}}}
+	route: {params: {item: WordType}}
 }
 
-export class DictionaryDetailView extends React.PureComponent<Props> {
-	static navigationOptions = ({navigation}: any) => {
-		return {
-			title: navigation.state.params.item.word,
-		}
+export const DetailNavigationOptions = (props: {
+	route: RouteProp<RootStackParamList, 'DictionaryDetail'>
+}): NativeStackNavigationOptions => {
+	let {item} = props.route.params
+	return {
+		title: item.word,
+		headerBackTitle: 'Dictionary',
 	}
+}
 
-	handleEditButtonPress = () => {
-		let item = this.props.navigation.state.params.item
-		this.props.navigation.navigate('DictionaryEditorView', {
-			word: item,
-		})
-	}
+export let DictionaryDetailView = (props: Props): JSX.Element => {
+	let [item] = React.useState<WordType>(props.route.params.item)
```

Comment:
```suggestion
	let item = props.route.params.item
```

Reply:
Are you wanting to avoid using state here?

Reply:
Yes, because we never do anything with it.